code/model/SubgoalPolicy.cpp

```diff
@@ -572,6 +571,15 @@ bool SubgoalPolicy::Init (void)
 	if (true == b_UseGoldLength)
 		LoadGoldLengthFile();
 
+	// RUN TESTS
+	if (IR_host != -1) {
+		if (false == o_IR.Connect ())
+			return false;
+
+		// Launch the test
+		TestQA();
+	}
+
```

Comment:
I run the tests after Init has initialized, so that the feature space is initialized